src/controllers/duplication-controller.js

```diff
@@ -0,0 +1,64 @@
+import { models } from '../models/init-models.js'
+import { Op } from 'sequelize'
+import sequelizeConn from '../util/db.js'
+
+export async function createRequest(req, res) {
+  const projectId = req.params.projectId
+  const remarks = req.body.remarks
+  const transfer = req.body.transfer
+
+  // if user chooses to transfer(boolean) onetime media
+  const onetimeAction = transfer ? 100 : 1
+
+  try {
+    const request = models.ProjectDuplicationRequest.build({
```

Comment:
Prefer `create` over `build` and `save`

Example in the code: `await models.TaxaXSpecimen.create(`